src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -685,4 +694,17 @@ private EmailWrapper getEmptyEmailAddressedToEmail(String recipient) {
         return email;
     }
 
+    private String getCoOwnersEmailsList(List<InstructorAttributes> instructors) {
```

Comment:
Commit https://github.com/TEAMMATES/teammates/pull/7169/commits/f6c7147dc13e8005ced3a8f70b2aa755ff6f11ae and Commit https://github.com/TEAMMATES/teammates/pull/7169/commits/93eb3a0c750dfc07f453a1f6ec5b9566a76b8f75